tools/pfe-tools/11ty/templates/install.njk

```diff
@@ -0,0 +1,23 @@
+{% set level = level or 2 %}
+{% set header = header or 'Installation' %}
+
+<section class="api band slots api-properties">
```

Comment:
this shouldn't have the api-properties class or the slots class, because it's about installation, not slots or properties